sections/builder.js

```diff
@@ -2439,6 +2439,20 @@ export default [
       }
     ]
   },
+  {
+    type: "method",
+    method: "optimizerHint",
+    example: ".optimizerHint(hint)",
+    description: "Add hints to the query using comment-like syntax `/*+ ... */`. MySQL and Oracle use this syntax for optimizer hints. Also various DB proxies and routers use this syntax to pass hints to alter their behavior. In other dialects the hints are ignored as simple comments.",
+    children: [
+      {
+        type: "runnable",
+        content: `
+          knex('accounts').where('userid', '=', 1).optimizerHint('hint()')
```

Comment:
`runnable` won't work before the change is released, as the scipt cant compile the query on the current knex version

Reply:
Can you use a real hint in the example?

Reply:
@kibertoad Added real MySQL hint example 7bc4df5